index.js

```diff
@@ -7,6 +7,8 @@ require('string.prototype.repeat');
 
 var CSS_ESCAPED_TAB = '\\9';
 
+var BLOCK_DISABLE = /postcss-increase-specificity disable/;
+var BLOCK_ENABLE = /postcss-increase-specificity enable/;
```

Comment:
Let's go with these names so we don't confuse blocks/rules and `RE` so we know it is a regex object,

 - `DISABLE_PLUGIN_RE`
 - `ENABLE_PLUGIN_RE`